packages/react/src/components/Tile/Tile.js

```diff
@@ -503,29 +504,31 @@ export class ExpandableTile extends Component {
     const {
       tabIndex,
       className,
+      expanded, // eslint-disable-line
       tileMaxHeight, // eslint-disable-line
       tilePadding, // eslint-disable-line
       handleClick, // eslint-disable-line
       tileCollapsedIconText, // eslint-disable-line
       tileExpandedIconText, // eslint-disable-line
+      onBeforeClick, // eslint-disable-line
       light,
       ...other
     } = this.props;
 
-    const { expanded } = this.state;
+    const { expanded: expandedState } = this.state;
```

Comment:
Maybe we could use this name to match our other conventions?

```suggestion
    const { expanded: isExpanded } = this.state;
```